exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 (c) Microsoft Corporation.  All rights reserved.
```

Comment:
Update year of copyright?

Reply:
Good point, have updated for all the files now. And Copyright is with OpenTelemetry, just like the dotnet exporter (https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/6d4fb62ee904582f2b0787ee5e02d227558c5359/src/OpenTelemetry.Exporter.Geneva/AssemblyInfo.cs#L2)